Antonio_Ramos_Documentation

```diff
@@ -381,3 +381,80 @@ Tab2 = Tab2.astype(convert_dict)
+  1. Spyder 4.0.1. (Installed through the anaconda navigator) 
+  2. arcpy (anaconda command prompt command "conda install -c esri arcpy")
+  3. plotnine (anaconda command prompt "conda install -c conda-forge plotnine") 
+  4. geopandas (anaconda command prmopt "conda install-c conda-forge geopandas") 
+- all moduels and packed installed successfully
+
+- when running scripts that import arcpy, i am getting an error that says "no module named 'arcpy')
+
+
+8/12/2020
+- created copy of programTrial3.6 environment on my personal computer to see if I can run the arcpy there
+
+- arcpy module succesfully loads in spyder on my personal computer
+
+- added the following code to ZCTA_Manufa_Trans.py script to recalculate feature extent..
+"#recalculate feature extent
+feature_class = geo_df
+arcpy.RecalculateFeatureClassExtent_management(feature_class)"
+
+- ZCTA_Manufa_Trans.py runs successfully with new code and exports shapefile (on my personal computer)
+
+- will check if the feature extent now matches the exent of the original data using the following code...
+"# Create a Describe object from the shapefile
+desc = arcpy.Describe(geo_df)
+
+# Print dataset properties
+print ("Dataset Type:" + desc.datasetType)
+
+extent = desc.Extent
+print ("Extent:")
+print ("  %s: %f, %s: %f, %s: %f, %s: %f" % ("XMin", extent.XMax, "XMax", extent.XMax, 
+                                            "YMin", extent.YMin, "YMax", extent.YMax))
+print ("MExtent: " + desc.MExtent)
+print ("ZExtent: " + desc.ZExtent)
+
+SR = desc.spatialReference
+print (SR.name)"
+
+- feature extent now matches original data
+
+8/13/2020
+
+- trying to now resolve why arcpy isn't being imported on the VM. I will remove programTrial3.6 and try again with new environment 3Stage_Local_Run_env, following same install procedures as I did on my personal computer
+
+- same problem occuring as well as other. There is probably an issue with our python instalation and I will ask ICT to remove and reinstall python, anaconda, and arcgis
+
+8/14/2020
+
+- discoverd why the feature extent is being changed when creating the shapefile for lab and ZCTA data. The command we are using to change the CRS is compressing the data in the shapefiles and not recalculating it with the new CRS. Hence, using the recalculate command stretches the compressed data back out.
+
+- will now use "geo_df.to_crs" instead of "geo_df.set_crs"
+
+- got the following error back "RuntimeError: b'no arguments in initialization list'"
+
+- my guess is this is related to the problems in the python installation because it is the same issue I was facing when I was having problems with the python installation last month
+
+8/16/2020
+- finally emailed ICT about python installation
+
+- investigating the block data information now to understand where it came from and how it was structured when it was made
+
+- unsure about what program was used to merge the two files that are refereced in Will's "What I Did" document
```

Comment:
It's possible that he did a join in ArcGIS -- if you can't find a program that does it, then assume this is the case. He started out doing a lot of operations in ArcGIS and was converting it all over to doing it programatically, but I can't guarantee he finished the conversion

Reply:
Ok I will see if I can find a way to programmatize this join.